src/util/requestWrapper.ts

```diff
@@ -68,9 +69,15 @@ export default class RequestUtil {
    * @private Make a HTTP request to a specific endpoint. Private endpoints are automatically signed.
    */
   async _call(method: Method, endpoint: string, params?: any): GenericAPIResponse {
-    const publicEndpoint = endpoint.startsWith('v2/public');
+    let publicEndpoint = false;
+    if(endpoint.startsWith('v2/public')){
+      publicEndpoint = true;
+    }
+    else if(endpoint.startsWith('public/linear/')){
+      publicEndpoint = true;
+    }
```

Comment:
Can you extract this to a function? Can put one outside the class, like:
```javascript
const isPublicEndpoint = (endpoint: string): boolean {
  if (endpoint.startsWith('v2/public')) {
    return true;
  }
  if (endpoint.startsWith('public/linear')) {
    return true;
  }
  return false;
}
```

That cleans this out a bit to:
```javascript
if (isPublicEndpoint(endpoint) === false) {
  ....
}
```